### PR TITLE
Changes / fixes for Issue #67

### PR DESCRIPTION
**Pull request by Orbisman**
Wrong line for XML event location in elements following DTD
- Added additional comment coverage in TestComments.java
- Added test per Issue #67 writeup to TestLocation.java
- Carriage line was not being checked so it was skipped, added check and call to skipCRLF(c) to advance line number.

### CODE REVIEW

File: src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java
Comment (cowtowncoder) [inline src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java:323]: I think that a better fix would be to change following line's test from `else if` to `if`, so both \r and \d would be checked.

File: src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java
Comment (cowtowncoder) [inline src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java:324]: @Orbisman Does this look ok? Happy to merge if so

File: src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java
Comment (cowtowncoder) [reply src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java]: Changed as per above suggestion.

**Comment by cowtowncoder** [comment]
Oh. One thing before merging: if I haven't yet asked for CLA, I'd need one from:

https://github.com/FasterXML/jackson/blob/master/contributor-agreement.pdf

(used for Woodstox as well as Jackson). It only needs to be sent once before the first contribution; after that good for all contributions. If you have sent one before (I tried checking but might have missed it) just let me know.

Otherwise it's easiest to print, fill & sign, scan/photo, send to `cla` at fasterxml dot com.

Once I have it I can proceed with merge.

Thank you for the PR; looking forward to merging it!

**Comment by Orbisman** [comment]
Your change looks good to me, I will work on the cla tomorrow night.

**Comment by cowtowncoder** [comment]
Perfect! Thank you @Orbisman.

**Comment by Orbisman** [comment]
CLA  sent...

**Comment by cowtowncoder** [comment]
CLA received, will merge.

**Review by cowtowncoder — APPROVED** [review]

**Pull request merged**
